### DISCUSSION TITLE
Where I can change automatic timeout parameter

### DISCUSSION THREAD

**Discussion by pfk-beta** [DeviceFarmer/stf#506] [Q&A] [answered]
Hello,

I'm getting automatic timeout after one hour, where I can change value for timeout parameter?

**Comment by denis99999** [comment, marked as answer]
@pfk-beta ,
https://github.com/DeviceFarmer/stf/issues/475

**Reply by ConfusedMerlin** [reply]
pardon me, would you mind telling me where to set the parameter mentioned in that answer if one does not use "stf local" but rather  every stf process as own call? I would opt for "stf provider", because it is usually responsible for most stuff happens to the connected devies.

**Reply by shri-0509** [reply]
you can pass group-timeout <value> attribute to your stf provider
